vividus-tests/src/main/resources/story/integration/HttpRequest.story

```diff
@@ -25,3 +25,8 @@ Then a JSON element by the JSON path '$.cookies' is equal to '{"vividus-cookie":
 Scenario: Verify HTTP cookies are cleared
 When I send HTTP GET to the relative URL '/cookies'
 Then a JSON element by the JSON path '$.cookies' is equal to '{}'
+
+Scenario: I wait for response code $responseCode for $duration duration retrying $retryTimes times $stepsToExecute
+When I wait for response code 200 for PT10S duration retrying 10 times
+|step                                                                                           |
```

Comment:
Please align the `|`  sign to the same indent  as next line to make code more beautiful